cli/project/dockerfiles_test.go

```diff
@@ -165,6 +165,13 @@ func TestGetBuildImageDockerfileTemplateEnterprise(t *testing.T) {
 
 	expLayers = `FROM centos:7
 
+### Fix Centos 8 EOL repo
```

Comment:
AFAIK usually either write everything in lower case as "centos" or "CentOS".

Reply:
fixed